apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/CreateCopyJobOptions.cs

```diff
@@ -43,6 +44,12 @@ public sealed class CreateCopyJobOptions : JobCreationOptions
         /// </summary>
         public CopyOperationType? OperationType { get; set; }
 
+        /// <summary>
```

Comment:
(As a person who doesn't have a context) This documentation doesn't help me understand what is its purpose.  Helper?

Reply:
Updated